src/pyspiffe/spiffe_id/trust_domain.py

```diff
@@ -1,6 +1,11 @@
+"""
+This module manage TrustDomain objects.
+"""
+
 from urllib.parse import urlparse, ParseResult
 from typing import Any, cast
 
+
 from pyspiffe.spiffe_id import SPIFFE_SCHEME
 
 EMPTY_DOMAIN_ERROR = 'Trust domain cannot be empty.'
```

Comment:
missing docstring.

Reply:
Thanks for the comments Glaucimar.
I've covered all of them. I added docstrings for these module variables. At some point we should check whether they need to be public to users of the module or not.